internal/entities/utils.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2024 Stacklok, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package entities contains logic relating to entity management
+package entities
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+
+	"github.com/stacklok/minder/internal/db"
+)
+
+// EntityFromIDs takes the IDs of the three known entity types and
+// returns a single ID along with the type of the entity.
+// This assumes that exactly one of the IDs is not equal to uuid.Nil
+func EntityFromIDs(
```

Comment:
This is smart, but I worry that the code might get fragile in the future. Since the caller already passes attributes of `interfaces.EvalStatusParams`, why not pass its `EntityType` attribute directly?

Reply:
The problem is that the way we refer to entities right now requires us to pass around a UUID field for each type of entity, so if another entity gets added we will need to add another parameter anyway.

I would love to come up with a more elegant way of managing entities in the codebase, but it's definitely a future problem.